picard/ui/options/maintenance.py

```diff
@@ -124,11 +124,10 @@ def load(self):
         orphan_options = file_options.difference(current_options)
 
         self.ui.option_counts.setText(
-            _("The configuration file currently contains %d option settings, %d which are unused.") % (
-                len(file_options),
-                len(orphan_options)
-            )
-        )
+            _("The configuration file currently contains %(totalcount)d option settings, %(unusedcount)d which are unused.") % {
```

Comment:
The string is pretty long, may be wrap:

```suggestion
            _("The configuration file currently contains %(totalcount)d option settings, "
                "%(unusedcount)d which are unused.") % {
```

Reply:
The entire block of code looks better when splitted